src/components/screens/sprint-view-screen.tsx

```diff
@@ -71,7 +68,8 @@ const SprintViewScreen = () => {
           }); 
           return totalMinutes;
         } catch (error) {
-          console.error(`Error fetching time entries for allocation ${allocation.id}: ${error}`);
+          const message: string = strings.formatString(strings.sprintRequestError.fetchAllocationError, (allocation.id||0).toString(), error as string).toString();
```

Comment:
Again the use of allocation.id || 0 even in the error message does not seem right. What does 0 mean, is there a way to say the allocation does not have an id rather than just 0?